packages/cozy-client/src/models/file.js

```diff
@@ -1,3 +1,5 @@
+import sub from 'date-fns/sub'
+import add from 'date-fns/add'
```

Comment:
`date-fns` is only a devDep currently. I don't know why... Also, devDep is currently 2.X while in our front app we use 1.X version. 

But since you added it to `fileModel` we should absolutely do something about that point. 
At first, it should be added to the `dep` section. But doing that we'll increase the size of the cozy-client lib. 

Maybe it should be a peerDep?

Reply:
UI has `date-fns@^1.28.5`, Drive has `date-fns@1.30.1`, MesPapiers (lib) has `date-fns@2.29.3` and MesPapiers (app) has `date-fns@2.23.0`.
So we have different major versions in the libs / apps that will consume this helpers.
Thus i don't think we can currently set it as a peer dependency.
I would be inclined to put in `dep` dependency in `cozy-client`, otherwise we would have to update the version in each consumer (perhaps we should).
Anyway, i will move the helpers in a `paper.js` file as you suggested.

Reply:
Nice let's have a duplicate package...

>  I would be inclined to put in dep dependency in cozy-client, otherwise we would have to update the version in each consumer (perhaps we should).

Yep, but we can't easily. See https://github.com/cozy/cozy-banks/pull/2522#issuecomment-1274636844 . We also have applications that remove duplicate package. It'll be a nightmare but I imagine this is what we need to do...

Reply:
on peut simplement
- passer date-fns en dep sur la dernière version
- mettre à jour la lib
- faire l'import profond

Reply:
Dans ce cas là, si la version de `date-fns` importe peu, alors une peerDep fera l'affaire.

Reply:
Je ré-ouvre ce point. Si on n'a pas besoin d'une version spécifique de date-fns alors peerDep. Sinon on a tous les bundlemon de toutes les app qui vont péter 🙏

Reply:
> Sans trop que je comprenne pourquoi, l'erreur qui m'a motivé à faire ce fix a disparu. J'ai quand même mis à jour date-fns et l'ai ajouté en dep. Aucun changement dans la lib entre les deux versions qui nous concerne apparemment.

Je ne faisais que reprendre ça pour dire d'ajouter en peerDep. Si on a besoin de add et de sub alors mettons en dep. Et laissons bundlemon nous rappeler à l'ordre.

Reply:
> est-ce que Yarn installe une peerDep s'il ne la trouve nulle part ?

Non, avec Yarn il faut passer par la commande `npx check-peer-dependencies --yarn --install` pour maj ou installer les perrDeps manquantes.

> Sur quoi on part ?
nit: Si c'est les fonctions les plus récentes, alors on pourrait être en peerDeps >= 2.9.0, et ainsi éviter les feux d'artifices ?

PS: Désolé d'intervenir si tardivement, cette PR était passée sous mon radar...

Reply:
Non pas de peerDep si on a besoin de date-fns en 2.9. On a plein d'app qui sont en 1.X sur date-fns et j'ai pas envie de me retrouver avec un BC à gérer là-dessus. 

Donc `dep`. Et le jour ou toutes les app seront passées à date-fns >= 2.9 alors on pourra regarder quoi faire. 


(ps: vu que Banks a de la dedup de package sur date-fns et que ça ne fait pas attention à la version, ça va être fun... J'espère qu'on ne passera pas dans ces méthodes depuis banks)

Reply:
Du coup ça reste en `dep` pour l'instant.